### PR TITLE
Docs: Add prefect-alert collection

### PR DESCRIPTION
**Pull request by khuyentran1401**
This PR added the [prefect-alert](https://github.com/khuyentran1401/prefect-alert) collection to the doc based on @ahuang11's suggestion.

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/collections/catalog/prefect-alert.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+collectionName: prefect-alert
+author: Prefect
```
Comment (tpdorsey) [inline docs/collections/catalog/prefect-alert.yaml:2]: Suggest making you the author until such time as this is moved to the prefect repos. Michael did that for his prefect-kv collection.

**Review by tpdorsey — APPROVED** [review]
Minor comment about attribution, but otherwise this looks great. Thanks

File: docs/collections/catalog/prefect-alert.yaml (hunk 1 above)
Comment (khuyentran1401) [reply docs/collections/catalog/prefect-alert.yaml]: Just updated the author. Thank you for reviewing!

**Comment by madkinsz** [comment]
@khuyentran1401 note this should include a `collection` tag in the future instead. `enhancement` is reserved for product changes.

**Pull request merged**